docker-compose.yaml

```diff
@@ -58,7 +58,7 @@ services:
     - POSTGRES_PASSWORD=maestro
 
   redis:
-    image: redis:3.2-alpine
+    image: redis:5.0.10-alpine
```

Comment:
Question: we're changing the version because our integration tests use the `5.0`, is that right?

Reply:
Yeap, exactly what we're doing. Checking our production redis versions, we're using 6.x redis databases.

Reply:
Yeap, exactly what we're doing. Checking our production redis versions, we're using 6.x redis databases.